### DISCUSSION TITLE
When I try to create a new field, gives me that error: Unknown argument `id_staff`. Available options are listed in green.

### DISCUSSION THREAD

**Discussion by PedroMartiniano** [prisma/prisma#21045] [Q&A]
### Question

I add a new field in my "course" model, that's the id_staff which is a foreign key for the id in the staff model, but when I try to create a new field, shows me that error: Unknown argument `id_staff`. Available options are listed in green.

here's my models:
```prisma
model Staff {
  id       String @id @default(uuid())
  cpf      String @unique
  email    String @unique
  username String @unique
  role     Role   @default(ADMIN)
  password String
  status   Int    @default(1)

  Course Course[]
  Module Module[]
}

model Course {
  id          String @id @default(uuid())
  name        String
  description String
  rating      Float  @default(0.0)
  qtdeRating  Int    @default(0)
  image       String
  price       Float
  id_staff    String
  status      Int    @default(1)

  staff    Staff      @relation(fields: [id_staff], references: [id])
  modules  Module[]
  purchase Purchase[]
}
```

and that's what i'm passing by my insomnia:
(ignorer the image field hahaha)

```js
{
	"name": "Curso de ruby",
	"description": "descrição ruby",
	"image": "abc",
	"price": 99.9,
	"id_staff": "87428c32-752d-4ae5-8807-9ad831d1a498"
}
```

**Comment by ludralph** [comment]
HI @PedroMartiniano 👋 

Thank you for raising this question.

I'm unable to reproduce this error as i can correctly create a `Course` record with `id_staff` as an argument.

```js
const res = await prisma.course.create({
  data: {
    "name": "Curso de ruby",
    "description": "descrição ruby",
    "image": "abc",
    "price": 99.9,
    "id_staff": 'd2b27f7c-6677-4591-8895-fdf2b987b635'
  }
})
```
This gives me the output
```js
{
  id: '8533e6e5-6a35-43c3-a274-9552d6c0a2f4',
  name: 'Curso de ruby',
  description: 'descrição ruby',
  rating: 0,
  qtdeRating: 0,
  image: 'abc',
  price: 99.9,
  id_staff: 'd2b27f7c-6677-4591-8895-fdf2b987b635',
  status: 1
}
```
Can you confirm that you are correctly parsing the `id_staff` field for creating the `Course` record? You may also want to share a reproduction in a repository so i can investigate further.

**Reply by ludralph** [reply]
Hey @PedroMartiniano

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!
